implementations/php-json-schema/bowtieJsonSchema.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+use Opis\JsonSchema\{
+    Validator
+};
+
+$STARTED = false;
+
+function start($request)
+{
+    if ($request->version !== 1) {
+        throw new Exception('Wrong version!');
+    }
+
+    $GLOBALS['STARTED'] = true;
+
+    $response = [
+        'ready' => true,
+        'version' => 1,
+        'implementation' => [
+            'language' => 'php',
+            'name' => 'json-schema',
+            'homepage' => 'https://opis.io/json-schema',
+            'issues' => 'https://github.com/opis/json-schema/issues',
+            'dialects' => [
+                'https://json-schema.org/draft/2020-12/schema',
+                'https://json-schema.org/draft-2019-09/schema',
+                'http://json-schema.org/draft-07/schema#',
+                'http://json-schema.org/draft-06/schema#',
+            ],
+            'os' => PHP_OS,
+            'os_version' => php_uname('r'),
+            'language_version' => PHP_VERSION,
+        ],
+    ];
+    return json_encode($response, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
+}
+
+function dialect($request)
+{
+    if (!$GLOBALS['STARTED']) {
+        throw new Exception('Not started!');
+    }
+    return json_encode(['ok' => false], JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
```

Comment:
Did you find the `Draft*` objects in this library?

They seem likely to be what you need here to properly support dialects.

E.g. it looks like here is how this is done in the library's test suite:

https://github.com/opis/json-schema/blob/c48df6d7089a45f01e1c82432348f2d5976f9bfb/tests/AbstractOfficialDraftTest.php#L69

where e.g. for Draft7 you for some reason pass the string '07':

https://github.com/opis/json-schema/blob/c48df6d7089a45f01e1c82432348f2d5976f9bfb/tests/OfficialDraft7Test.php#L27

If you don't find a mapping within the library that maps between dialect URIs (e.g. http://json-schema/whatever/draft7) and those magic strings (07) then you probably want to make one here and do that lookup and store the value as a global.